server/src/test/scala/RoutesTests.scala

```diff
@@ -0,0 +1,18 @@
+package server
+
+import org.scalatest.FunSuite
+import akka.http.scaladsl.testkit.ScalatestRouteTest
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594746744/issues/source?bid=3301982&fileBranchId=3306761#l4)